test/lib/instructions-parser.spec.ts

```diff
@@ -60,6 +60,16 @@ describe("getPackagesFromRunInstructions", () => {
         );
       },
     );
+    test("given an image built with --build-args, expect ARG values to be stripped from RUN instruction", () => {
```

Comment:
Tiny: I think the flag is actually `--build-arg` without the s

Reply:
Good spot, thanks